// @hosterai/provider-client/dist/enums/language.enum.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.LanguageEnum = void 0;
var LanguageEnum;
(function (LanguageEnum) {
    LanguageEnum["LANG_AA"] = "aa";
    LanguageEnum["LANG_AF"] = "af";
    LanguageEnum["LANG_AK"] = "ak";
    LanguageEnum["LANG_SQ"] = "sq";
    LanguageEnum["LANG_AM"] = "am";
    LanguageEnum["LANG_AR"] = "ar";
    LanguageEnum["LANG_AN"] = "an";
    LanguageEnum["LANG_HY"] = "hy";
    LanguageEnum["LANG_AS"] = "as";
    LanguageEnum["LANG_AV"] = "av";
    LanguageEnum["LANG_AE"] = "ae";
    LanguageEnum["LANG_AY"] = "ay";
    LanguageEnum["LANG_AZ"] = "az";
    LanguageEnum["LANG_BM"] = "bm";
    LanguageEnum["LANG_BA"] = "ba";
    LanguageEnum["LANG_EU"] = "eu";
    LanguageEnum["LANG_BE"] = "be";
    LanguageEnum["LANG_BN"] = "bn";
    LanguageEnum["LANG_BH"] = "bh";
    LanguageEnum["LANG_BI"] = "bi";
    LanguageEnum["LANG_BS"] = "bs";
    LanguageEnum["LANG_BR"] = "br";
    LanguageEnum["LANG_BG"] = "bg";
    LanguageEnum["LANG_MY"] = "my";
    LanguageEnum["LANG_CA"] = "ca";
    LanguageEnum["LANG_CH"] = "ch";
    LanguageEnum["LANG_CE"] = "ce";
    LanguageEnum["LANG_NY"] = "ny";
    LanguageEnum["LANG_ZH"] = "zh";
    LanguageEnum["LANG_CV"] = "cv";
    LanguageEnum["LANG_KW"] = "kw";
    LanguageEnum["LANG_CO"] = "co";
    LanguageEnum["LANG_CR"] = "cr";
    LanguageEnum["LANG_HR"] = "hr";
    LanguageEnum["LANG_CS"] = "cs";
    LanguageEnum["LANG_DA"] = "da";
    LanguageEnum["LANG_DV"] = "dv";
    LanguageEnum["LANG_NL"] = "nl";
    LanguageEnum["LANG_DZ"] = "dz";
    LanguageEnum["LANG_EN"] = "en";
    LanguageEnum["LANG_EO"] = "eo";
    LanguageEnum["LANG_ET"] = "et";
    LanguageEnum["LANG_EE"] = "ee";
    LanguageEnum["LANG_FO"] = "fo";
    LanguageEnum["LANG_FJ"] = "fj";
    LanguageEnum["LANG_FI"] = "fi";
    LanguageEnum["LANG_FR"] = "fr";
    LanguageEnum["LANG_FF"] = "ff";
    LanguageEnum["LANG_GL"] = "gl";
    LanguageEnum["LANG_KA"] = "ka";
    LanguageEnum["LANG_DE"] = "de";
    LanguageEnum["LANG_EL"] = "el";
    LanguageEnum["LANG_GN"] = "gn";
    LanguageEnum["LANG_GU"] = "gu";
    LanguageEnum["LANG_HT"] = "ht";
    LanguageEnum["LANG_HA"] = "ha";
    LanguageEnum["LANG_HE"] = "he";
    LanguageEnum["LANG_HZ"] = "hz";
    LanguageEnum["LANG_HI"] = "hi";
    LanguageEnum["LANG_HO"] = "ho";
    LanguageEnum["LANG_HU"] = "hu";
    LanguageEnum["LANG_IA"] = "ia";
    LanguageEnum["LANG_ID"] = "id";
    LanguageEnum["LANG_IE"] = "ie";
    LanguageEnum["LANG_GA"] = "ga";
    LanguageEnum["LANG_IG"] = "ig";
    LanguageEnum["LANG_IK"] = "ik";
    LanguageEnum["LANG_IO"] = "io";
    LanguageEnum["LANG_IS"] = "is";
    LanguageEnum["LANG_IT"] = "it";
    LanguageEnum["LANG_IU"] = "iu";
    LanguageEnum["LANG_JA"] = "ja";
    LanguageEnum["LANG_JV"] = "jv";
    LanguageEnum["LANG_KL"] = "kl";
    LanguageEnum["LANG_KN"] = "kn";
    LanguageEnum["LANG_KR"] = "kr";
    LanguageEnum["LANG_KS"] = "ks";
    LanguageEnum["LANG_KK"] = "kk";
    LanguageEnum["LANG_KM"] = "km";
    LanguageEnum["LANG_KI"] = "ki";
    LanguageEnum["LANG_RW"] = "rw";
    LanguageEnum["LANG_KY"] = "ky";
    LanguageEnum["LANG_KV"] = "kv";
    LanguageEnum["LANG_KG"] = "kg";
    LanguageEnum["LANG_KO"] = "ko";
    LanguageEnum["LANG_KU"] = "ku";
    LanguageEnum["LANG_KJ"] = "kj";
    LanguageEnum["LANG_LA"] = "la";
    LanguageEnum["LANG_LB"] = "lb";
    LanguageEnum["LANG_LG"] = "lg";
    LanguageEnum["LANG_LI"] = "li";
    LanguageEnum["LANG_LN"] = "ln";
    LanguageEnum["LANG_LO"] = "lo";
    LanguageEnum["LANG_LT"] = "lt";
    LanguageEnum["LANG_LU"] = "lu";
    LanguageEnum["LANG_LV"] = "lv";
    LanguageEnum["LANG_GV"] = "gv";
    LanguageEnum["LANG_MK"] = "mk";
    LanguageEnum["LANG_MG"] = "mg";
    LanguageEnum["LANG_MS"] = "ms";
    LanguageEnum["LANG_ML"] = "ml";
    LanguageEnum["LANG_MT"] = "mt";
    LanguageEnum["LANG_MI"] = "mi";
    LanguageEnum["LANG_MR"] = "mr";
    LanguageEnum["LANG_MH"] = "mh";
    LanguageEnum["LANG_MN"] = "mn";
    LanguageEnum["LANG_NA"] = "na";
    LanguageEnum["LANG_NV"] = "nv";
    LanguageEnum["LANG_ND"] = "nd";
    LanguageEnum["LANG_NE"] = "ne";
    LanguageEnum["LANG_NG"] = "ng";
    LanguageEnum["LANG_NB"] = "nb";
    LanguageEnum["LANG_NN"] = "nn";
    LanguageEnum["LANG_NO"] = "no";
    LanguageEnum["LANG_II"] = "ii";
    LanguageEnum["LANG_NR"] = "nr";
    LanguageEnum["LANG_OC"] = "oc";
    LanguageEnum["LANG_OJ"] = "oj";
    LanguageEnum["LANG_CU"] = "cu";
    LanguageEnum["LANG_OM"] = "om";
    LanguageEnum["LANG_OR"] = "or";
    LanguageEnum["LANG_OS"] = "os";
    LanguageEnum["LANG_PA"] = "pa";
    LanguageEnum["LANG_PI"] = "pi";
    LanguageEnum["LANG_FA"] = "fa";
    LanguageEnum["LANG_PL"] = "pl";
    LanguageEnum["LANG_PS"] = "ps";
    LanguageEnum["LANG_PT"] = "pt";
    LanguageEnum["LANG_QU"] = "qu";
    LanguageEnum["LANG_RM"] = "rm";
    LanguageEnum["LANG_RN"] = "rn";
    LanguageEnum["LANG_RO"] = "ro";
    LanguageEnum["LANG_RH"] = "rh";
    LanguageEnum["LANG_RU"] = "ru";
    LanguageEnum["LANG_SA"] = "sa";
    LanguageEnum["LANG_SC"] = "sc";
    LanguageEnum["LANG_SD"] = "sd";
    LanguageEnum["LANG_SE"] = "se";
    LanguageEnum["LANG_SM"] = "sm";
    LanguageEnum["LANG_SG"] = "sg";
    LanguageEnum["LANG_SR"] = "sr";
    LanguageEnum["LANG_GD"] = "gd";
    LanguageEnum["LANG_SN"] = "sn";
    LanguageEnum["LANG_SI"] = "si";
    LanguageEnum["LANG_SK"] = "sk";
    LanguageEnum["LANG_SL"] = "sl";
    LanguageEnum["LANG_SO"] = "so";
    LanguageEnum["LANG_ST"] = "st";
    LanguageEnum["LANG_ES"] = "es";
    LanguageEnum["LANG_SU"] = "su";
    LanguageEnum["LANG_SW"] = "sw";
    LanguageEnum["LANG_SS"] = "ss";
    LanguageEnum["LANG_SV"] = "sv";
    LanguageEnum["LANG_TA"] = "ta";
    LanguageEnum["LANG_TE"] = "te";
    LanguageEnum["LANG_TG"] = "tg";
    LanguageEnum["LANG_TH"] = "th";
    LanguageEnum["LANG_TI"] = "ti";
    LanguageEnum["LANG_BO"] = "bo";
    LanguageEnum["LANG_TK"] = "tk";
    LanguageEnum["LANG_TL"] = "tl";
    LanguageEnum["LANG_TN"] = "tn";
    LanguageEnum["LANG_TO"] = "to";
    LanguageEnum["LANG_TR"] = "tr";
    LanguageEnum["LANG_TS"] = "ts";
    LanguageEnum["LANG_TT"] = "tt";
    LanguageEnum["LANG_TW"] = "tw";
    LanguageEnum["LANG_TY"] = "ty";
    LanguageEnum["LANG_UG"] = "ug";
    LanguageEnum["LANG_UK"] = "uk";
    LanguageEnum["LANG_UR"] = "ur";
    LanguageEnum["LANG_UZ"] = "uz";
    LanguageEnum["LANG_VE"] = "ve";
    LanguageEnum["LANG_VI"] = "vi";
    LanguageEnum["LANG_VO"] = "vo";
    LanguageEnum["LANG_WA"] = "wa";
    LanguageEnum["LANG_CY"] = "cy";
    LanguageEnum["LANG_WO"] = "wo";
    LanguageEnum["LANG_FY"] = "fy";
    LanguageEnum["LANG_XH"] = "xh";
    LanguageEnum["LANG_YI"] = "yi";
    LanguageEnum["LANG_YO"] = "yo";
    LanguageEnum["LANG_ZA"] = "za";
    LanguageEnum["LANG_ZU"] = "zu";
    LanguageEnum["LANG_ABKHAZ"] = "ab";
    LanguageEnum["LANG_AFAR"] = "aa";
    LanguageEnum["LANG_AFRIKAANS"] = "af";
    LanguageEnum["LANG_AKAN"] = "ak";
    LanguageEnum["LANG_ALBANIAN"] = "sq";
    LanguageEnum["LANG_AMHARIC"] = "am";
    LanguageEnum["LANG_ARABIC"] = "ar";
    LanguageEnum["LANG_ARAGONESE"] = "an";
    LanguageEnum["LANG_ARMENIAN"] = "hy";
    LanguageEnum["LANG_ASSAMESE"] = "as";
    LanguageEnum["LANG_AVARIC"] = "av";
    LanguageEnum["LANG_AVESTAN"] = "ae";
    LanguageEnum["LANG_AYMARA"] = "ay";
    LanguageEnum["LANG_AZERBAIJANI"] = "az";
    LanguageEnum["LANG_BAMBARA"] = "bm";
    LanguageEnum["LANG_BASHKIR"] = "ba";
    LanguageEnum["LANG_BASQUE"] = "eu";
    LanguageEnum["LANG_BELARUSIAN"] = "be";
    LanguageEnum["LANG_BENGALI_BANGLA"] = "bn";
    LanguageEnum["LANG_BIHARI"] = "bh";
    LanguageEnum["LANG_BISLAMA"] = "bi";
    LanguageEnum["LANG_BOSNIAN"] = "bs";
    LanguageEnum["LANG_BRETON"] = "br";
    LanguageEnum["LANG_BULGARIAN"] = "bg";
    LanguageEnum["LANG_BURMESE"] = "my";
    LanguageEnum["LANG_CATALAN"] = "ca";
    LanguageEnum["LANG_CHAMORRO"] = "ch";
    LanguageEnum["LANG_CHECHEN"] = "ce";
    LanguageEnum["LANG_NYANJA"] = "ny";
    LanguageEnum["LANG_CHINESE"] = "zh";
    LanguageEnum["LANG_CHUVASH"] = "cv";
    LanguageEnum["LANG_CORNISH"] = "kw";
    LanguageEnum["LANG_CORSICAN"] = "co";
    LanguageEnum["LANG_CREE"] = "cr";
    LanguageEnum["LANG_CROATIAN"] = "hr";
    LanguageEnum["LANG_CZECH"] = "cs";
    LanguageEnum["LANG_DANISH"] = "da";
    LanguageEnum["LANG_DIVEHI"] = "dv";
    LanguageEnum["LANG_DUTCH"] = "nl";
    LanguageEnum["LANG_DZONGKHA"] = "dz";
    LanguageEnum["LANG_ENGLISH"] = "en";
    LanguageEnum["LANG_ESPERANTO"] = "eo";
    LanguageEnum["LANG_ESTONIAN"] = "et";
    LanguageEnum["LANG_EWE"] = "ee";
    LanguageEnum["LANG_FAROESE"] = "fo";
    LanguageEnum["LANG_FIJIAN"] = "fj";
    LanguageEnum["LANG_FINNISH"] = "fi";
    LanguageEnum["LANG_FRENCH"] = "fr";
    LanguageEnum["LANG_FULA_FULAH"] = "ff";
    LanguageEnum["LANG_GALICIAN"] = "gl";
    LanguageEnum["LANG_GEORGIAN"] = "ka";
    LanguageEnum["LANG_GERMAN"] = "de";
    LanguageEnum["LANG_GREEK"] = "el";
    LanguageEnum["LANG_GUARAN\u00CD"] = "gn";
    LanguageEnum["LANG_GUJARATI"] = "gu";
    LanguageEnum["LANG_HAITIAN"] = "ht";
    LanguageEnum["LANG_HAUSA"] = "ha";
    LanguageEnum["LANG_HEBREW"] = "he";
    LanguageEnum["LANG_HERERO"] = "hz";
    LanguageEnum["LANG_HINDI"] = "hi";
    LanguageEnum["LANG_HIRI"] = "ho";
    LanguageEnum["LANG_HUNGARIAN"] = "hu";
    LanguageEnum["LANG_INTERLINGUA"] = "ia";
    LanguageEnum["LANG_INDONESIAN"] = "id";
    LanguageEnum["LANG_INTERLINGUE"] = "ie";
    LanguageEnum["LANG_IRISH"] = "ga";
    LanguageEnum["LANG_IGBO"] = "ig";
    LanguageEnum["LANG_INUPIAQ"] = "ik";
    LanguageEnum["LANG_IDO"] = "io";
    LanguageEnum["LANG_ICELANDIC"] = "is";
    LanguageEnum["LANG_ITALIAN"] = "it";
    LanguageEnum["LANG_INUKTITUT"] = "iu";
    LanguageEnum["LANG_JAPANESE"] = "ja";
    LanguageEnum["LANG_JAVANESE"] = "jv";
    LanguageEnum["LANG_KALAALLISUT"] = "kl";
    LanguageEnum["LANG_KANNADA"] = "kn";
    LanguageEnum["LANG_KANURI"] = "kr";
    LanguageEnum["LANG_KASHMIRI"] = "ks";
    LanguageEnum["LANG_KAZAKH"] = "kk";
    LanguageEnum["LANG_KHMER"] = "km";
    LanguageEnum["LANG_KIKUYU"] = "ki";
    LanguageEnum["LANG_KINYARWANDA"] = "rw";
    LanguageEnum["LANG_KYRGYZ"] = "ky";
    LanguageEnum["LANG_KOMI"] = "kv";
    LanguageEnum["LANG_KONGO"] = "kg";
    LanguageEnum["LANG_KOREAN"] = "ko";
    LanguageEnum["LANG_KURDISH"] = "ku";
    LanguageEnum["LANG_KWANYAMA"] = "kj";
    LanguageEnum["LANG_LATIN"] = "la";
    LanguageEnum["LANG_LUXEMBOURGISH"] = "lb";
    LanguageEnum["LANG_GANDA"] = "lg";
    LanguageEnum["LANG_LIMBURGISH"] = "li";
    LanguageEnum["LANG_LINGALA"] = "ln";
    LanguageEnum["LANG_LAO"] = "lo";
    LanguageEnum["LANG_LITHUANIAN"] = "lt";
    LanguageEnum["LANG_LUBA"] = "lu";
    LanguageEnum["LANG_LATVIAN"] = "lv";
    LanguageEnum["LANG_MANX"] = "gv";
    LanguageEnum["LANG_SKOPJE"] = "mk";
    LanguageEnum["LANG_MALAGASY"] = "mg";
    LanguageEnum["LANG_MALAY"] = "ms";
    LanguageEnum["LANG_MALAYALAM"] = "ml";
    LanguageEnum["LANG_MALTESE"] = "mt";
    LanguageEnum["LANG_M\u0100ORI"] = "mi";
    LanguageEnum["LANG_MARATHI"] = "mr";
    LanguageEnum["LANG_MARSHALLESE"] = "mh";
    LanguageEnum["LANG_MONGOLIAN"] = "mn";
    LanguageEnum["LANG_NAURU"] = "na";
    LanguageEnum["LANG_NAVAJO"] = "nv";
    LanguageEnum["LANG_NDEBELE_NORTHERN"] = "nd";
    LanguageEnum["LANG_NEPALI"] = "ne";
    LanguageEnum["LANG_NDONGA"] = "ng";
    LanguageEnum["LANG_NORWEGIAN"] = "nb";
    LanguageEnum["LANG_NUOSU"] = "ii";
    LanguageEnum["LANG_NDEBELE"] = "nr";
    LanguageEnum["LANG_OCCITAN"] = "oc";
    LanguageEnum["LANG_OJIBWA"] = "oj";
    LanguageEnum["LANG_OLD_CHURCH_SLAVONIC"] = "cu";
    LanguageEnum["LANG_OROMO"] = "om";
    LanguageEnum["LANG_ORIYA"] = "or";
    LanguageEnum["LANG_OSSETIAN"] = "os";
    LanguageEnum["LANG_PANJABI"] = "pa";
    LanguageEnum["LANG_P\u0100LI"] = "pi";
    LanguageEnum["LANG_FARSI"] = "fa";
    LanguageEnum["LANG_POLISH"] = "pl";
    LanguageEnum["LANG_PUSHTO"] = "ps";
    LanguageEnum["LANG_PORTUGUESE"] = "pt";
    LanguageEnum["LANG_QUECHUA"] = "qu";
    LanguageEnum["LANG_ROMANSH"] = "rm";
    LanguageEnum["LANG_KIRUNDI"] = "rn";
    LanguageEnum["LANG_ROMANIAN"] = "ro";
    LanguageEnum["LANG_ROTHONGUA"] = "rh";
    LanguageEnum["LANG_RUSSIAN"] = "ru";
    LanguageEnum["LANG_SANSKRIT"] = "sa";
    LanguageEnum["LANG_SARDINIAN"] = "sc";
    LanguageEnum["LANG_SINDHI"] = "sd";
    LanguageEnum["LANG_SAMI"] = "se";
    LanguageEnum["LANG_SAMOAN"] = "sm";
    LanguageEnum["LANG_SANGO"] = "sg";
    LanguageEnum["LANG_SERBIAN"] = "sr";
    LanguageEnum["LANG_GAELIC"] = "gd";
    LanguageEnum["LANG_SHONA"] = "sn";
    LanguageEnum["LANG_SINHALA"] = "si";
    LanguageEnum["LANG_SLOVAK"] = "sk";
    LanguageEnum["LANG_SLOVENE"] = "sl";
    LanguageEnum["LANG_SOMALI"] = "so";
    LanguageEnum["LANG_SOTHO"] = "st";
    LanguageEnum["LANG_SPANISH"] = "es";
    LanguageEnum["LANG_SUNDANESE"] = "su";
    LanguageEnum["LANG_SWAHILI"] = "sw";
    LanguageEnum["LANG_SWATI"] = "ss";
    LanguageEnum["LANG_SWEDISH"] = "sv";
    LanguageEnum["LANG_TAMIL"] = "ta";
    LanguageEnum["LANG_TELUGU"] = "te";
    LanguageEnum["LANG_TAJIK"] = "tg";
    LanguageEnum["LANG_THAI"] = "th";
    LanguageEnum["LANG_TIGRINYA"] = "ti";
    LanguageEnum["LANG_TIBETAN"] = "bo";
    LanguageEnum["LANG_TURKMEN"] = "tk";
    LanguageEnum["LANG_TAGALOG"] = "tl";
    LanguageEnum["LANG_TSWANA"] = "tn";
    LanguageEnum["LANG_TONGA"] = "to";
    LanguageEnum["LANG_TURKISH"] = "tr";
    LanguageEnum["LANG_TSONGA"] = "ts";
    LanguageEnum["LANG_TATAR"] = "tt";
    LanguageEnum["LANG_TWI"] = "tw";
    LanguageEnum["LANG_TAHITIAN"] = "ty";
    LanguageEnum["LANG_UYGHUR"] = "ug";
    LanguageEnum["LANG_UKRAINIAN"] = "uk";
    LanguageEnum["LANG_URDU"] = "ur";
    LanguageEnum["LANG_UZBEK"] = "uz";
    LanguageEnum["LANG_VENDA"] = "ve";
    LanguageEnum["LANG_VIETNAMESE"] = "vi";
    LanguageEnum["LANG_VOLAP\u00DCK"] = "vo";
    LanguageEnum["LANG_WALLOON"] = "wa";
    LanguageEnum["LANG_WELSH"] = "cy";
    LanguageEnum["LANG_WOLOF"] = "wo";
    LanguageEnum["LANG_FRISIAN"] = "fy";
    LanguageEnum["LANG_XHOSA"] = "xh";
    LanguageEnum["LANG_YIDDISH"] = "yi";
    LanguageEnum["LANG_YORUBA"] = "yo";
    LanguageEnum["LANG_ZHUANG"] = "za";
    LanguageEnum["LANG_ZUL"] = "zu";
})(LanguageEnum = exports.LanguageEnum || (exports.LanguageEnum = {}));
