// @hosterai/provider-client/dist/client/client.service.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Client = void 0;
const axios_1 = require("axios");
const addon_client_1 = require("./addon.client");
const issue_client_1 = require("./issue.client");
const payment_client_1 = require("./payment.client");
const product_client_1 = require("./product.client");
const task_client_1 = require("./task.client");
const template_client_1 = require("./template.client");
class Client {
    constructor(api_url, service_provider_token) {
        this.api_url = api_url;
        this.service_provider_token = service_provider_token;
        this.addons = new addon_client_1.AddonClient(axios_1.default, this.api_url, this.service_provider_token);
        this.tasks = new task_client_1.TaskClient(axios_1.default, this.api_url, this.service_provider_token);
        this.templates = new template_client_1.TemplateClient(axios_1.default, this.api_url, this.service_provider_token);
        this.products = new product_client_1.ProductClient(axios_1.default, this.api_url, this.service_provider_token);
        this.issues = new issue_client_1.IssueClient(axios_1.default, this.api_url, this.service_provider_token);
        this.payments = new payment_client_1.PaymentClient(axios_1.default, this.api_url, this.service_provider_token);
    }
}
exports.Client = Client;
